src/ride/track.h

```diff
@@ -173,7 +173,7 @@ typedef struct {
 	uint8 track_spine_colour[4];					// 0x60
 	uint8 track_rail_colour[4];						// 0x64
 	uint8 track_support_colour[4];					// 0x68
-	uint32 var_6C;									// some flags
+	uint32 var_6C;									// some extra flags. (1 << 31) was used to mark the TD6 as a Six Flags ride.
```

Comment:
Do we really need to make an enum for a flag we don't use anymore? Duncan only asked to clarify it in a comment, that's why I put it here.

Renaming var_6C to flags2 would be a good idea, I think.